### PR TITLE
HPC: Add basic HDF5 test

### PR DESCRIPTION
**Pull request by schlad**
The openmpi got renamed to openmpi1

- Related ticket: https://progress.opensuse.org/issues/137138
- Verification run:

### CODE REVIEW

**Comment by czerw** [comment]
`HPC: Ugly fix for rpm re-name` is not telling much what we ugly fixing. Please consider something like `HPC: Rename openmpi to openmpi1` .

File: tests/hpc/mpi_hdf5.pm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use lockapi;
+use utils;
+use registration;
+use Utils::Logging 'export_logs';
+use hpc::formatter;
+
+sub run ($self) {
+    record_info('TODO', 'Write simple test');
```
Comment (czerw) [inline tests/hpc/mpi_hdf5.pm:21]: I would prefer to track TODO outside of code.

Comment (schlad) [reply tests/hpc/mpi_hdf5.pm:21]: sure, this is more of a comment to myself for now; I simply want to see easily and quickly that the test run arrived at this point ;-)

**Comment by schlad** [comment]
> `HPC: Ugly fix for rpm re-name` is not telling much what we are ugly fixing. Please consider something like `HPC: Rename openmpi to openmpi1` .

anyway that fix to change openmpi to openmpi1 won't work for the current mpi_master since the naming convention for various pieces is different and varies from no "1", to "_1" and just "rpm1" and adding more if-else to account for that would be even more crazy.

After playing with hdf5, I think I will propose a PR with new approach for those HPC_LIB tests where we have hdf as a module loaded by scheduler based on values passed in HPC_LIB;
mpi_master will stay untouched until we agree on this new way of testing stuff like hdf5, papie and alike; if we agree and merge the thing I will be de-coupling things we see in mpi_master and will be moving them to separate modules mpi_papi.pm, mpi_openblas.pm etc.

**Pull request closed without merging**